theforeman.org/yaml/includes/pulpcore_versions.yaml.inc

```diff
@@ -1,4 +1,5 @@
 - '3.16'
 - '3.17'
 - '3.18'
-- '3.21'
\ No newline at end of file
+- '3.21'
+- '3.22'
```

Comment:
Please ensure a line end is here.

Reply:
Ack

Reply:
It still shows the symbol that there's no line ending.

Reply:
Had to use vim, lol

Reply:
I don't know why you keep marking it as resolved, but it's still not there.